lisa/sut_orchestrator/azure/common.py

```diff
@@ -1344,6 +1346,7 @@ def check_or_create_resource_group(
     cloud: Cloud,
     resource_group_name: str,
     location: str,
+    managed_by: str,
```

Comment:
Give a default value `""`. So it doesn't need to be provided, if it's not needed.